<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableOperateItem</class>
 <widget class="QWidget" name="TableOperateItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,3,2,0,0">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">width:90px;
height:30px;
max-width:90px;
max-height:30px;
min-width:60px;
min-height:30px;</string>
     </property>
     <property name="text">
      <string>通道设置</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="styleSheet">
      <string notr="true">width:60px;
height:30px;
max-width:90px;
max-height:30px;
min-width:60px;
min-height:30px;</string>
     </property>
     <property name="text">
      <string>下放</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_3">
     <property name="minimumSize">
      <size>
       <width>62</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>62</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">width:60px;
height:30px;
max-width:90px;
max-height:30px;
min-width:60px;
min-height:30px;</string>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
